enhancements/installer/single-node-installation.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Single node installation
+authors:
+  - "@eranco74"
+reviewers:
+  - "@romfreiman"
+  - "@hardys"
+  - "@crawford"
+  - "@markmc"
+approvers:
+  - "@hardys"
+  - "@crawford"
+  - "@markmc"
+creation-date: yyyy-mm-dd
+last-updated: yyyy-mm-dd
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+see-also:
+  - "https://github.com/openshift/enhancements/pull/302"
+  - "https://github.com/openshift/installer/pull/3978"
+replaces:
+  - "/enhancements/that-less-than-great-idea.md"
+superseded-by:
+  - "/enhancements/our-past-effort.md"
+---
+
+
+# Single node installation
+
+Add a new `create aio-config` command to `openshift-installer` which
```

Comment:
I am not wild about abbreviated command names, although I am ~ok with shorter aliases.  I'd rather address long-command-name concerns with auto-complete scripts ;).  Can we make this `create single-node-config` or some such?

Reply:
Hmm, it's also not clear to me why you can't just use the existing `create ignition-configs` with an `install-config.yaml` requesting `replicas: 1` for the control plane and `replicas: 0` for compute.  Why does this need a new subcommand?

Reply:
The new sub-command is required since we want a new installation flow that allow installing the node without having an auxiliary node (bootstrap) just an rhcos + ignition.